src/App.js

```diff
@@ -5,16 +5,18 @@ import { Body } from "./Routes/homeBody/body.jsx";
 import { ShopNow } from "./Routes/Products/shopNow.jsx";
 import { Footer } from "./Components/Footer/footer.jsx";
 import { NavBar } from "./Components/Navbar/navbar.jsx";
+import { CartCard } from "./Components/CartCard/cartCard.jsx";
 function App() {
   return (
     <div className="App">
        <NavBar />
       <Routes>
         <Route path="/" element={<Body />} />
         <Route path="/shopNow" element={<ShopNow />} />
+        <Route path="/cart" element={<CartCard />} />
```

Comment:
Shouldn't it go to the `Cart` route? instead of `CartCard`.
maybe a naming issue.